src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin: 0 auto;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: 0.3s all;
+
+  &:hover {
+    transform: scale(1.2);
+    cursor: pointer;
```

Comment:
It is not necessary to set the parameter for the `cursor` on each `hover`, it is enough to set it for the class.